Header header

int32[] vector_pixels